src/server/generators/attribute.js

```diff
@@ -1,3 +1,5 @@
+import { isUndefined } from '../../shared/typeof'
```

Comment:
Could we introduce an alias (`~` or so) so we can get rid of relative paths?

Can do that in a separate PR as well

Reply:
Please do as I am not sure how this works actually. Is it a babel feature? Or is it that symlink trick?

Reply:
It's a webpack feature ☺️